frontend/src/index.pug

```diff
@@ -174,13 +174,25 @@ html
               .summary-charts__title--index {{ i+1 }}
               .summary-charts__title--repo {{ repo[0].repoName }}
               .summary-charts__title--contribution.mini-font(title="Total contribution of group") [{{ getGroupTotalContribution(repo) }} lines]
+              template(v-if="isMergeGroup")
+                a(
+                  v-bind:href="getRepoLink(repo[0])", target="_blank",
+                  title="click to view author's repo"
```

Comment:
Should the link say "click to view **group's** repo", since we are grouping by repos?

This is what I see:
![image](https://user-images.githubusercontent.com/35621759/72951657-29ab4600-3dca-11ea-8a69-76ba70685c39.png)

Reply:
Yes, my bad. Thought I changed it. Will update it.